alignak/bin/alignak_reactionner.py

```diff
@@ -63,12 +63,12 @@
 import imp
 try:
     imp.find_module('android')
-    is_android = True
+    IS_ANDROID = True
 except ImportError:
-    is_android = False
+    IS_ANDROID = False
 
 
-if is_android is not None:
+if IS_ANDROID is not None:
```

Comment:
Not linked with C0103, but IS_ANDROID is never None because we have defined 4 lignes before True or False